app/controllers/content/index.js

```diff
@@ -261,23 +281,11 @@ export default Ember.Controller.extend(Analytics, {
                 node: this.get('node'),
             };
 
-            this.get('currentUser').load()
-                .then(user => {
-                    if (user) {
-                        const userId = user.id;
-                        authors.forEach((author) => {
-                            if (author.get('userId') === userId) {
-                                userIsContrib = true;
-                            }
-                        });
-                    }
-                    if (!userIsContrib) {
-                        Ember.get(this, 'metrics').invoke('trackSpecificCollection', 'Keen', keenPayload); // Sends event to Keen if logged-in user is not a contributor
-                    }
-                })
-                .catch(() => {
-                    Ember.get(this, 'metrics').invoke('trackSpecificCollection', 'Keen', keenPayload); // Sends event to Keen for non-authenticated user
-                });
+            if (!this.get('userIsContrib')) {
+                Ember.get(this, 'metrics').invoke('trackSpecificCollection', 'Keen', keenPayload); // Sends event to Keen if logged-in user is not a contributor
+            } else {
+                Ember.get(this, 'metrics').invoke('trackSpecificCollection', 'Keen', keenPayload); // Sends event to Keen for non-authenticated user
```

Comment:
This if statement does the same thing in both branches. Based on the original code, seems like it should just be the first one -- the new `userIsContrib` will be false for non-authenticated users, too.

Reply:
Oh yup